src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURES = 5;
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int randomNumberOfFigure = random.nextInt(NUMBER_OF_FIGURES);
+        switch (randomNumberOfFigure) {
+            case 0:
+                Square square = new Square();
+                return square.getRandomFigure();
```

Comment:
```suggestion
                return getRandomSquare()
```
To make your code easier to read - lets create private methods that return each private figure.
Also you should randomize parameters of figure here, and set those values in that method. This logic should not be in each figures class